deps/uv/src/win/fs-event.c

```diff
@@ -225,11 +225,10 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   }
 
   if (!handle->buffer) {
-    handle->buffer = (char*)_aligned_malloc(uv_directory_watcher_buffer_size,
-                                            sizeof(DWORD));
+    handle->buffer = (char*)uv__malloc(uv_directory_watcher_buffer_size);
```

Comment:
@saghul IIUC, here the call to `_aligned_malloc` would be replaced by a call to `malloc`, is it intentional and what is the impact of such a change?